src/views/Orders.vue

```diff
@@ -418,6 +420,16 @@ export default defineComponent({
     },
     viewShipToStoreOrders() {
       this.$router.push({ path: '/ship-to-store-orders' })
+    },
+    getPickersForOrder(pickers: [string]){
+      let pickersInfo = '';
+      pickers.forEach((picker: any, index: number) => {
+        pickersInfo += picker.split('/')[1];
+        if(pickers.length != index + 1){
```

Comment:
Simplify this logic, as we should not check against the index number and then update the picker info string.